### ISSUE TITLE
Update All Error

### ISSUE THREAD

**Issue by drewbz1089** [theypsilon/Update_All_MiSTer#100]
Downloaded the latest update all and put in "scripts" folder to run update. Getting error before update takes off saying "Temporary failure in name resolution.  Any ideas?

Thanks!

-Louis

**Comment by theypsilon** [comment]
Hey, thanks for reporting. This is probably a network error. But in any case, please share your logs at Scripts/.config/update_all/update_all.log and Scripts/.config/downloader/downloader.log

**Comment by TruREALigion** [comment]
been having so much grief ever since the 2.0 release.  I tried manually updating CACERT.pem.  I tried manually replacing the /linux/linux.img.  I now get strange errors saying Internet connectivity not working.  I confirmed I can ping out so its not a network error.

_Linux Version: 221224
WARNING: cacert file at "/etc/ssl/certs/cacert.pem" seems to be wrong!
         Return Code: 6
Removing /etc/ssl/certs/cacert.pem

Downloading new cacert file...
ERROR: Download failed! 6

Attempting again in 10 seconds...

WARNING: cacert file at "/etc/ssl/certs/cacert.pem" seems to be wrong!
         Return Code: 6
Removing /etc/ssl/certs/cacert.pem

Downloading new cacert file...
ERROR: Download failed! 6

Attempting again in 10 seconds...

WARNING: cacert file at "/etc/ssl/certs/cacert.pem" seems to be wrong!
         Return Code: 6
Removing /etc/ssl/certs/cacert.pem

Downloading new cacert file...
ERROR: Download failed! 6
ERROR: Couldn't load certificates.

Please, reboot your system and try again._

**Comment by theypsilon** [comment]
Have you tried with another SD card? It is not normal that cacert.pem is 0 bytes. The script wouldn't cause that.

As an alternative, you can use the insecure launcher: https://raw.githubusercontent.com/theypsilon/Update_All_MiSTer/master/update_all_insecure.sh

Put that one in Scripts and run it instead, but don't delete update_all.sh

**Comment by TruREALigion** [comment]
This would be the next step (that I am trying to avoid).

my other MiSTer setups are fine.  Perhaps it is truly an SD Card issue.

**Comment by theypsilon** [comment]
Please reopen if error persists.

**Comment by TruREALigion** [comment]
more to this.  I was able to get the update_all script to work when "hardwired" through ethernet.  I should have mentioned that I had this MiSTer on wifi using a TP-Link T3U Plus.  

After fully updating MiSTer, I plugged back the Wifi to see if it resolves itself but I get random results:

- typical return code 6
- Time Zone errors
- is the network connected

I do not believe it is an issue with my wifi (Ubiquiti) but I will do some troubleshooting.  I will also see if there is any Firmware updates for the TP-Link T3U
